src/app/legacy/containers/Navigation/index.test.jsx

```diff
@@ -113,4 +116,82 @@ describe('Navigation Container', () => {
       expect(href).toEqual(navItem.url);
     });
   });
+
+  describe('View and click tracking', () => {
+    const scrollEventTrackingData = {
+      componentName: 'scrollable-navigation',
+    };
+
+    const dropdownEventTrackingData = {
+      componentName: 'dropdown-navigation',
+    };
+
+    const clickTrackerSpy = jest
+      .spyOn(clickTracking, 'default')
+      .mockImplementation();
+
+    it('should call the view tracking hook when on scrollable navigation', () => {
+      const viewTrackerSpy = jest.spyOn(viewTracking, 'default');
+      render(<Navigation />, {
+        bbcOrigin: 'https://www.test.bbc.co.uk',
+        id: 'c0000000000o',
+        isAmp: true,
+        pageType: ARTICLE_PAGE,
+        service: 'news',
+        statusCode: 200,
+        pathname: '/news',
+      });
+      expect(viewTrackerSpy).toHaveBeenCalledWith(scrollEventTrackingData);
+    });
+
+    it('should call the view tracking hook when on dropdown navigation', () => {
+      const viewTrackerSpy = jest.spyOn(viewTracking, 'default');
+      render(<Navigation />, {
+        bbcOrigin: 'https://www.test.bbc.co.uk',
+        id: 'c0000000000o',
+        isAmp: true,
+        pageType: ARTICLE_PAGE,
+        service: 'news',
+        statusCode: 200,
+        pathname: '/news',
+      });
+      expect(viewTrackerSpy).toHaveBeenCalledWith(dropdownEventTrackingData);
+    });
+
+    it('should call the click tracking hook when scrollable navigation is clicked', () => {
+      clickTrackerSpy.mockRestore();
+
+      const { container } = render(<Navigation />, {
+        bbcOrigin: 'https://www.test.bbc.co.uk',
+        id: 'c0000000000o',
+        isAmp: true,
+        pageType: ARTICLE_PAGE,
+        service: 'news',
+        statusCode: 200,
+        pathname: '/news',
+      });
+
+      fireEvent.click(container);
+
+      expect(container.onclick).toBeTruthy();
+    });
+
+    it('should call the click tracking hook when dropdown navigation is clicked', () => {
```

Comment:
How do we force a click on the dropdown navigation? Do we need to change the viewport? I can't see the difference between this test and the one above... 🤔

Reply:
Good question, I prematurely put this up for review!

Reply:
Marking as resolved, I have removed the click test for dropdown navigation. As discussed, we will test this with a cypress test!